monkey/infection_monkey/ransomware/ransomware_payload_builder.py

```diff
@@ -0,0 +1,44 @@
+from infection_monkey.ransomware import readme_utils
+from infection_monkey.ransomware.file_selectors import ProductionSafeTargetFileSelector
+from infection_monkey.ransomware.in_place_file_encryptor import InPlaceFileEncryptor
+from infection_monkey.ransomware.ransomware_payload import RansomwarePayload
+from infection_monkey.ransomware.targeted_file_extensions import TARGETED_FILE_EXTENSIONS
+from infection_monkey.telemetry.messengers.batching_telemetry_messenger import (
+    BatchingTelemetryMessenger,
+)
+from infection_monkey.telemetry.messengers.legacy_telemetry_messenger_adapter import (
+    LegacyTelemetryMessengerAdapter,
+)
+from infection_monkey.utils.bit_manipulators import flip_bits
+
+EXTENSION = ".m0nk3y"
+CHUNK_SIZE = 4096 * 24
+
+
+def build_ransomware_payload(config: dict):
```

Comment:
This looks more like a constructor. I'm not sure how would I build a different payload with this. I think this method should take a class/module as a parameter and use `_build_file_encryptor`, `_build_file_selector` and `_build_telemetry_messenger` defined in that class to create the payload object.